src/UKHO.S100PermitService.Common/Services/PermitService.cs

```diff
@@ -108,11 +128,12 @@ private static List<Products> GetProductsList()
             return productsList;
         }
 
-        private static List<ProductKeyServiceRequest> ProductKeyServiceRequest(List<HoldingsServiceResponse> holdingsServiceResponse) =>
-             holdingsServiceResponse.SelectMany(x => x.Cells.Select(y => new ProductKeyServiceRequest
-             {
-                 ProductName = y.CellCode,
-                 Edition = y.LatestEditionNumber
-             })).ToList();
+        private static List<ProductKeyServiceRequest> ProductKeyServiceRequest(
+            List<HoldingsServiceResponse> holdingsServiceResponse) =>
```

Comment:
Generic List(T) argument should be IEnumerable(T)

Reply:
Resolved